src/script/script_instance.cpp

```diff
@@ -691,7 +691,7 @@ void ScriptInstance::LoadOnStack(ScriptData *data)
 {
 	ScriptObject::ActiveInstance active(this);
 
-	if (data == nullptr) return;
+	if (this->IsDead() || data == nullptr) return;
```

Comment:
Not necessarily for this PR, but shouldn't the `ScriptData` be released somewhere when the script is dead?

Reply:
It is released on any case after the call to `ScriptInstance::LoadOnStack()`